### ISSUE TITLE
IDEA: Using own Google API credentials

### ISSUE THREAD

**Issue by gnanet** [arielscarpinelli/oldtube#4]
To avoid over usage of your API Account a simple how-to about generating API-key and OAuth 2.0 credentials would help, like:

1. one needs to create a [google account](https://myaccount.google.com/)
2. [activate developer access](https://console.developers.google.com/apis)
3. [enable YouTube Data API v3 and generate credentials](https://developers.google.com/identity/protocols/OAuth2ForDevices#prerequisites)
4. then replace the values of **youTubeApiKey** **oAuthClientId** **oAuthClientSecret** in _src/js/config.js_

**Comment by arielscarpinelli** [comment]
Feel free to PR the README about this. I started this project for helping people without such technical skill.

**Comment by the4anoni** [comment]
Or just add option to enter apinkeus in app settings :) That would be MUCH simpler

**Comment by ConstantinCreations** [comment]
Yes please! I generated my own keys but I'm having problems converting this project to a USB loadable app.
